usagov/usagov-2021
# USAGOV-1367: USAGOV-1367 - Delay of CEO no results error message

https://cm-jira.usa.gov/browse/USAGOV-1367

## Description
Making the CEO results hidden by default, and then showing them when the API responds with results

## Type of Changes


## Testing Instructions
* Go to the path: /elected-officials
* Put in your address, submit the form
* Expect to the a loading/spinner to show in the page (briefly) before the results show

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
